edit/gatekeeper-constraint.vue

```diff
@@ -201,7 +202,7 @@ export default {
     },
     onTabChanged({ tab }) {
       // This is necessary to force the yamlEditor to adjust the size once it has space to fill.
-      if (tab.name === 'parameters' & this.$refs.yamlEditor?.refresh) {
+      if (tab.name === 'parameters' && this.$refs.yamlEditor?.refresh) {
```

Comment:
Surprised eslint config doesn't have something enabled for unexpected bitwise &/|/^...